store/theme.store.ts

```diff
@@ -1,30 +1,28 @@
-import type {} from "@redux-devtools/extension"; // required for devtools typing
-import { create } from "zustand";
-import { devtools, persist } from "zustand/middleware";
+import type {} from "@redux-devtools/extension"
+// required for devtools typing
+import { create } from "zustand"
+import { devtools, persist } from "zustand/middleware"
 
 interface themeState {
-  theme: "light" | "dark";
-  userInteracted: boolean;
-  toggleTheme: () => void;
+  theme: "light" | "dark" | undefined
```

Comment:
I think `dark`and `light`types could be defined globally in our[ lib/types dir](https://github.com/danskernesdigitalebibliotek/dpl-go/tree/main/lib/types)